constants/items.py

```diff
@@ -295,3 +295,25 @@
     "Marvel Shoes",
     "Exp. Egg",
 ]
+
+## TRASH LIST https://docs.google.com/spreadsheets/d/1Cit5Xl_TCBPFI4q1NEVQhSYGH1wKY9st_1yee3lzCP8/edit#gid=0
+
+TRASH_WEAPONS = [
+    "Dirk", "MithrilKnife", "Guardian", "MithrilBlade", "RegalCutlass", "Crystal", "Ogre Nix", "Mithril Pike",
+    "Stout Spear", "Gold Lance", "Partisan", "Imperial", "Kodachi", "Hardened", "Ashura", "Kotetsu", "Forged",
+    "Aura", "Strato", "Poison Rod", "Punisher", "Gravity Rod", "MetalKnuckle", "Mithril Claw", "Kaiser",
+    "Flail", "Morning Star", "Full Moon", "Boomerang", "Rising Sun", "Cards", "Darts", "Chocobo Brsh", "DaVinci Brsh"
```

Comment:
Please keep the DaVinci Brsh, Poison Rod, Gravity Rod, Gold Lance, Partisan, and Ogre Nix -- they've all niche use

Reply:
Despite my feelings about Ogre Nix, I will make this change